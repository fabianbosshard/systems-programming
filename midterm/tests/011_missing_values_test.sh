exec "$1" format=@b,@a delim=" " <<EOF
Lime
Papaya 7
EOF
