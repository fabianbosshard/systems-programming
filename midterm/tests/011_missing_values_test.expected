,Lime
7,Papaya
